Added Browse NuGet steps in "Get started with WebView2 in Win32 apps" (#1622)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/get-started/win32.md

---

> title: Get started with WebView2 in Win32 apps

Reviewer: copied from offline: "Add note saying: if you are not seeing proper search results, please check the nuget package source location listed in Options->NuGet Package Manager->Package Sources , and ensure theres a rouce [route?] pointing to https://api.nuget.org/v3/index.json dsfa"

Author: Proposed -- WARNING: If you do not see the expected search results, check the NuGet source location. Click Tools > Options > NuGet Package Manager > Package Sources. Make sure that in Package sources there is a nuget.com route pointing to  https://api.nuget.org/v3/index.json. If not, enter nuget.com in the Name text box and https://api.nuget.org/v3/index.json in the Source text box. Then click Update and OK.

       Manage NuGet packages
    :::image-end:::

    > [!WARNING]

>     > If you do not see the expected search results in the following steps, check the NuGet source location. Click **Tools** > **Options** > **NuGet Package Manager** > **Package Sources**. Make sure that in **Package sources** there is a **nuget.com** route pointing to https://api.nuget.org/v3/index.json. If not, enter `nuget.com` in the **Name** text box and `https://api.nuget.org/v3/index.json` in the **Source** text box. Then click **Update** and **OK**.

Reviewer: ```suggestion
    > If you don't see the expected search results in the following steps, check the NuGet source location. Click **Tools** > **Options** > **NuGet Package Manager** > **Package Sources**. Make sure that in **Package sources** there is a **nuget.com** route pointing to `https://api.nuget.org/v3/index.json`. If **Package sources** doesn't contain that route, enter `nuget.com` in the **Name** text box and `https://api.nuget.org/v3/index.json` in the **Source** text box. Then click **Update** and **OK**.
```
wrapping url so it doesn't render as a clickable link

    :::image-end:::

1.  Install the Windows Implementation Library.

>     1.  In the **Nuget** window click the **Browse** tab.

Reviewer: Global: change to NuGet cap'zn per https://www.nuget.org/

Author: fixing

        :::image-end:::

1.  Install the WebView2 SDK.

>     1.  In the **Nuget** window click the **Browse** tab.

Reviewer: NuGet?

Author: yes
